veevavault/1.3/modules/ROOT/pages/veevavault-connector-examples.adoc

```diff
@@ -441,23 +466,24 @@ output application/x-www-form-urlencoded
 }
 ----
 +
-Note: Configure the above parameters according to your requirements to invoke the Vault REST API.
+[NOTE]
+You must configure the above parameters according to your requirements to invoke the Vault REST API.
```

Comment:
Is there a way to add "according to your requirements" to the steps rather than separately in a NOTE...especially one that _follows_ the steps (theoretically already taken)?